.tool-versions

```diff
@@ -1,2 +1,3 @@
-erlang 22.3
-elixir 1.10.2
+elixir 1.11.2
+erlang 23.1.4
+rust 1.46.0
```

Comment:
I think rust 1.44 is used in builder images

Reply:
ah ah ah, lets bump 1.44 in builder